src/frontend/src/pages/TeamsPage/TeamMembersView.tsx

```diff
@@ -0,0 +1,123 @@
+import PageBlock from '../../layouts/PageBlock';
```

Comment:
also i dont like all our files named "view" and want to move away from that. how about something like `TeamMembersPageBlock`